packages/alchemy/package.json

```diff
@@ -47,6 +47,10 @@
   },
   "dependencies": {
     "@alchemy/aa-core": "^3.0.0-alpha.5",
+    "@turnkey/http": "^2.6.2",
```

Comment:
can we add nohoist config for these so that other 3rd  party libs don't get hoisted to the root workspace?

Reply:
Not familiar with that, do you have a link to what that looks like / does?